examples/SkeletalPushout.g

```diff
@@ -17,8 +17,7 @@ iota2 := EmbeddingOfFinSets( N2, M );
 #! |2| ↪ |5|
 Display( iota2 );
 #! { 0, 1 } ⱶ[ 0 .. 1 ]→ { 0,..., 4 }
-D := [ iota1, iota2 ];
-#! [ |3| ↪ |5|, |2| ↪ |5| ]
```

Comment:
There is information here that is not displayed anymore.

Reply:
`iota1` and `iota2` are already printed in lines 10 and 16.

Reply:
Now I see them, thanks.